feature-libs/organization/package.json

```diff
@@ -22,25 +21,15 @@
   "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/organization",
   "schematics": "./schematics/collection.json",
   "peerDependencies": {
+    "@angular-devkit/schematics": "^10.1.0",
     "@angular/common": "^10.1.0",
     "@angular/core": "^10.1.0",
-    "@angular-devkit/schematics": "^10.1.0",
     "@angular/router": "^10.1.0",
-    "bootstrap": "^4.0",
-    "rxjs": "^6.6.0",
     "@spartacus/core": "3.0.0",
+    "@spartacus/schematics": "3.0.0",
```

Comment:
By doing this, is it possible to read "live" changes from the projects/schematics during development?

Reply:
I didn't changed anything here 👀 
I'll check, but it's just prettier reorder.